exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/Metadata.java

```diff
@@ -918,18 +916,22 @@ public void setMax(Object max) {
     @JsonProperty public ConcurrentHashMap<ColumnTypeMetadata_v2.Key, ColumnTypeMetadata_v2> columnTypeInfo;
     @JsonProperty List<ParquetFileMetadata_v2> files;
     @JsonProperty List<String> directories;
-    @JsonProperty String drillVersion;
```

Comment:
I agree regarding future similar issues. So I returned this field to `ParquetTableMetadataBase` in the new commit. 
But to determine that the parquet file is new and definitely with correct date values we have a new flag in parquet metadata "is.date.correct = true". Thus, it is more easy to determine corrupted date values and there is no need to wait the end of release to merge these changes.